packages/react-charting/.npmignore

```diff
@@ -37,3 +37,4 @@ visualtests
 !lib-amd
 !dist
 docs
+UnitTests
```

Comment:
this entry should not be needed as *.test.* is already present in this file

Reply:
This PR should be good to merge once this comment is resolved. For remaining comments, Balakrishna can address in a different PR.

Reply:
Hi @AtishayMsft we are checking this in now and will incorporate this change in separate PR.